src/Subscriptions/SubscriptionPurchase.php

```diff
@@ -306,15 +306,15 @@ public function getAutoResumeTime(): ?Time
     /**
      * @return IntroductoryPriceInfo
      */
-    public function getIntroductoryPriceInfo(): IntroductoryPriceInfo
+    public function getIntroductoryPriceInfo(): ?IntroductoryPriceInfo
```

Comment:
What about returning `null` if the `introductoryPriceInfo` is null and create the `IntroductoryPriceInfo` object otherwise!